test/e2e_node/e2e_node_suite_test.go

```diff
@@ -80,6 +80,7 @@ func registerNodeFlags(flags *flag.FlagSet) {
 	// It is hard and unnecessary to deal with the complexity inside the test suite.
 	flags.BoolVar(&framework.TestContext.NodeConformance, "conformance", false, "If true, the test suite will not start kubelet, and fetch system log (kernel, docker, kubelet log etc.) to the report directory.")
 	flags.BoolVar(&framework.TestContext.PrepullImages, "prepull-images", true, "If true, prepull images so image pull failures do not cause test failures.")
+	flags.BoolVar(&framework.TestContext.RestartKubelet, "restart-kubelet", true, "If true, restart Kubelet unit when the process is killed.")
```

Comment:
this seems fine as long as sig node are OK with the change.
@kubernetes/sig-node-pr-reviews 

> Adding `--restart-kubelet` flag on E2E Node test suite

please explain what the flag does in the release note too.

Reply:
looks like the flag was already approved here:
https://github.com/kubernetes/kubernetes/pull/96775#pullrequestreview-541122956